libindy_vdr/tests/nym.rs

```diff
@@ -253,6 +272,240 @@ mod send {
+            .build_nym_request(
+                &trustee.did,
+                &identity.did, // Non self-cert identifier
+                Some(identity.verkey.to_string()),
+                Some(ALIAS.to_string()),
+                Some(ROLE.to_string()),
+                None,
+                Some(2), // Claim Self-cert version 2
+            )
+            .unwrap();
+
+        let _err = helpers::sign_and_send_request(&trustee, &pool, &mut nym_request).unwrap_err();
+    }
+
+    #[rstest]
+    fn test_pool_send_nym_request_with_non_self_cert_did_works(
+        pool: TestPool,
+        trustee: Identity,
+        non_self_cert_identity: Identity,
+    ) {
+        // Send NYM
+        let identity = non_self_cert_identity;
+        println!("DID: {}", identity.did);
+        let mut nym_request = pool
+            .request_builder()
+            .build_nym_request(
+                &trustee.did,
+                &identity.did, // Non self-cert identifier
+                Some(identity.verkey.to_string()),
+                Some(ALIAS.to_string()),
+                Some(ROLE.to_string()),
+                None,
+                None,
+            )
+            .unwrap();
+
+        let _err = helpers::sign_and_send_request(&trustee, &pool, &mut nym_request).unwrap();
+    }
+
+    #[rstest]
+    fn test_pool_send_nym_request_with_version_0_works(
+        pool: TestPool,
+        trustee: Identity,
+        identity: Identity,
+    ) {
+        // Send NYM
+        let mut nym_request = pool
+            .request_builder()
+            .build_nym_request(
+                &trustee.did,
+                &identity.did, // Self-cert version 1 identifier
+                Some(identity.verkey.to_string()),
+                Some(ALIAS.to_string()),
+                Some(ROLE.to_string()),
+                None,
+                Some(0), // Claim Self-cert version 0
+            )
+            .unwrap();
+
+        let nym_response =
+            helpers::sign_and_send_request(&trustee, &pool, &mut nym_request).unwrap();
+
+        // Get NYM
+        let get_nym_request = pool
+            .request_builder()
+            .build_get_nym_request(None, &identity.did, None, None)
+            .unwrap();
+
+        let response = pool
+            .send_request_with_retries(&get_nym_request, &nym_response)
+            .unwrap();
+
+        let expected_data = json!({
+            "identifier": &trustee.did,
+            "dest": &identity.did,
+            "verkey": &identity.verkey,
+            "role": role_to_code(Some(String::from(ROLE))).unwrap(),
+            "version": 0,
+        });
+
+        let data: GetNymResultV1 = serde_json::from_str(
+            helpers::get_response_data(&response)
+                .unwrap()
+                .as_str()
+                .unwrap(),
+        )
+        .unwrap();
+
+        assert_eq!(expected_data, serde_json::to_value(data).unwrap());
+    }
+
+    // FIXME: Uncomment as soon as indy-utils is updated
```

Comment:
indy-utils did receive an update. Can we uncomment this of do we have a to keep it commented?